cluster/saltbase/salt/fluentd-es/fluentd-es.yaml

```diff
@@ -5,20 +5,20 @@ metadata:
 spec:
   containers:
   - name: fluentd-elasticsearch
-    image: gcr.io/google_containers/fluentd-elasticsearch:1.5
-    env:
-    - name: "FLUENTD_ARGS"
-      value: "-qq"
+    image: gcr.io/google_containers/fluentd-elasticsearch:1.6
+    args:
+    - -qq
     volumeMounts:
     - name: varlog
-      mountPath: /varlog
-    - name: containers
+      mountPath: /var/log
```

Comment:
I realize that this should work, but at the same time it seems risky to shadow a mounted volume over a directory that exists in the container image and is used by default by some many different processes. Are there any best practices around this?

It certainly feels safer to put it somewhere else, something like /mnt/var/log, but I can live with this is you've done this sort of thing with containers before.